app/config/locale/translations/templates/np.email.auth.recovery.tpl

```diff
@@ -0,0 +1,15 @@
+<p>
+    नमस्कार {{name}},
+</p>
+<p>
+   तपाईंको {{project}}को  पासवर्ड रिसेट गर्नका लागि यो लिंक अनुसरण गर्नुहोस्।
```

Comment:
remove extra space from को  पासवर्ड =>को पासवर्ड

Reply:
ohh thanks i will do that.

Reply:
@spielers please check now. if any error then i would happily resolve again. Thanks!